src/components/settings/Settings.h

```diff
@@ -245,4 +246,4 @@ namespace Pinetime {
       void SaveSettingsToFile();
     };
   }
-}
+}
```

Comment:
Please don't remove the last empty line at the end of the files. Try to keep the commits clean.

```suggestion
}

```